### PR TITLE
Update library.js

### PR DESCRIPTION
**Pull request by atomoc**

### CODE REVIEW

File: library.js [hunk 1]
```diff
@@ -2,20 +2,19 @@
...
-		async = module.parent.require('async'),
-		winston = module.parent.require('winston');
+		async = module.parent.require('async');
 
 	var constants = Object.freeze({
 		'name': 'Facebook',
 		'admin': {
 			'route': '/plugins/sso-facebook',
-			'icon': 'fa-facebook-square'
+			'icon': 'fa fa-facebook-square'
```
Comment (julianlam) [inline library.js]: The `fa` is added automatically.

File: library.js [hunk 2]
```diff
@@ -31,51 +30,53 @@
...
 	};
 
-	Facebook.login = function(fbid, name, email, picture, callback) {
+	Facebook.login = function(fbid, name, username, email, picture, callback) {
+		if (!email) {
+			email = name.replace(' ', '_')
+				.toLowerCase() 
+					+ '@users.noreply.facebook.com';
+		}
+
```
Comment (julianlam) [inline library.js]: What is the rationale for this change rather than using the logic that was removed arond line 60?

Comment (atomoc) [inline library.js]: I just copied from the sso-yandex. It is necessary to return the string with email.

File: library.js (hunk 1 above)
Comment (atomoc) [inline library.js]: Ok, thanks!

**Comment by julianlam** [comment]
I ended up changing the way Facebook retrieves its settings, so this PR isn't applicable anymore. Thanks though!
